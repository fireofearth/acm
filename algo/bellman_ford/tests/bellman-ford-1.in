9 11
1
1 2 10
2 3 10
2 4 -1
4 5 -1
5 2 -1
2 6 1
6 7 1
7 2 1
2 8 -1
8 9 -1
9 2 -1
