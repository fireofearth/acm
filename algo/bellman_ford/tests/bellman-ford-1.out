cycle detected
4 5 2
